lib/stack/roles/index.js

```diff
@@ -130,7 +130,7 @@ export function Role (http, data) {
      * import * as contentstack from '@contentstack/management'
      * const client = contentstack.client()
      *
-     * client.stack().role().findAll()
+     * client.stack().role().fetchAll()
```

Comment:
We are considering naming the method find() to retrieve all available data, while for fetching specific single data, we will use fetch().